src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
Looks like I'm late to the party, but my take here is a vote against the inclusion of such line.

> To sum up, these things are all part of the specific local setup that we don't dictate. I, too, am an IntelliJ fan. However, adding stuff to the codebase (comments or otherwise) for the express purpose of conforming to a certain IDE is what I'm against. I'll only have it if we specifically decide to recommend one IDE and its default inspection profile over the others, then consistently stick to it throughout the codebase.

This line by @whipermr5 pretty much sums up my stand. It is difficult to enforce even _one_ IDE profile; you'd have noticed by now that the defaults of any IDE lack many good features of (at least Java) languages. My own Eclipse inspection profile has many more additional rules on top of the default ones, and I'm not sure how many Eclipse violations are detected out there, but I got 245 right now (many of which I'm itching to fix). I'm not saying that you should throw away good IDE features, but unless they are general enough (e.g other IDEs will also complain anyway; CheckStyle/... will complain anyway), I don't think we have a ground to include any IDE-specific annotations around. It is like saying "I [a person] don't understand this code, let's add a comment here" while the majority of people out there have no problem understanding it.